index.html

```diff
@@ -35,22 +35,20 @@
       <section class="mdl-hero">
         <div class="m-content">
           <div class="m-title">Secure and fast microVMs for serverless computing</div>
-          <a href="https://github.com/firecracker-microvm/firecracker" class="css-button2 sta-hoverOFF">
-            View on GitHub
-          </a>
+          <a href="https://github.com/firecracker-microvm/firecracker" class="css-button2 sta-hoverOFF">View on GitHub</a>
```

Comment:
Since we mostly work this file via code editors (it's not autogenerated), it helps both readability and edit-ability to nest text within elements on their own line. My preference is for keeping that it as-is (also applies to subsequent of this kind edits below).